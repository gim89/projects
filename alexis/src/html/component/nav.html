<nav class="nav">
  <div class="container nav__container">
    <a class="nav__logo">Alexis</a>
    <ul class="nav__list">
      <li class="nav__item">
        <a href="#about" class="nav__link nav__link--hover nav__link--active">About Us</a>
      </li>
      <li class="nav__item">
        <a href="#focus" class="nav__link nav__link--hover nav__link--active">Our Focus</a>
      </li>
      <li class="nav__item">
        <a href="#portfolio" class="nav__link nav__link--hover nav__link--active">Portfolio</a>
      </li>
      <li class="nav__item">
        <a href="#team" class="nav__link nav__link--hover nav__link--active">Our Team</a>
      </li>
      <li class="nav__item">
        <a href="#steps" class="nav__link nav__link--hover nav__link--active">Work Steps</a>
      </li>
      <li class="nav__item">
        <a href="#pricing" class="nav__link nav__link--hover nav__link--active">Pricing</a>
      </li>
      <li class="nav__item">
        <a href="#testimonials" class="nav__link nav__link--hover nav__link--active">Testimonials</a>
      </li>
      <li class="nav__item">
        <a href="#blog" class="nav__link nav__link--hover nav__link--active">Blog</a>
      </li>
      <li class="nav__item">
        <a href="#contact" class="nav__link nav__link--hover nav__link--active">Contact</a>
      </li>
    </ul>
  </div>
</nav>