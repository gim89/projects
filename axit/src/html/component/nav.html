<nav class="nav">
  <div class="nav__container container">
    <a href=".head" class="nav__logo">
      <span class="nav__span">ax</span>
      it
    </a>
    <ul class="nav__list">
      <li class="nav__item">
        <a href="#feature-one" class="nav__link nav__link--hover nav__link--active">Features</a>
      </li>
      <li class="nav__item">
        <a href="#process" class="nav__link nav__link--hover nav__link--active">About</a>
      </li>
      <li class="nav__item">
        <a href="#pricing" class="nav__link nav__link--hover nav__link--active">Pricing</a>
      </li>
      <li class="nav__item">
        <a href="#testimonial" class="nav__link nav__link--hover nav__link--active">Reviews</a>
      </li>
      <li class="nav__item">
        <a href="#contact" class="nav__link nav__link--hover nav__link--active">Contact</a></li>
    </ul>
  </div>

</nav>