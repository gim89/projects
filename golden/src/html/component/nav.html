<nav class="nav">
  <a href="" class="nav__logo">Golden</a>
  <ul class="nav__list">
    <li class="nav__item nav__item--hover">
      <a href="#home" class="nav__link">home</a>
    </li>
    <li class="nav__item nav__item--hover">
      <a href="#services" class="nav__link">services</a>
    </li>
    <li class="nav__item nav__item--hover">
      <a href="#portfolio" class="nav__link">portfolio</a>
    </li>
    <li class="nav__item nav__item--hover">
      <a href="#about" class="nav__link">about</a>
    </li>
    <li class="nav__item nav__item--hover">
      <a href="#contact" class="nav__link">contact</a>
    </li>
  </ul>
</nav>